### DISCUSSION TITLE
vscode 插件

### DISCUSSION THREAD

**Discussion by hacker0limbo** [flomoapp/3rd-party-tools#39] [分享创造]
基于 flomo 提供的 API 写了一个插件: https://marketplace.visualstudio.com/items?itemName=Limboer.vscode-flomo

支持通过 `input box` 方式发送比较简短的想法, 也支持通过文档的方式将文档中的文字记录为想法

源码地址为: https://github.com/hacker0limbo/vscode-flomo

欢迎提 issue 和 pr!

**Comment by lightory** [comment]
真棒，看来 Code 高亮支持需要考虑起来了。

**Reply by hacker0limbo** [reply]
也谢谢浮墨提供的 API
